test/gtest/ucp/test_ucp_request.cc

```diff
@@ -377,7 +377,7 @@ class test_proto_reset : public ucp_test {
                         unsigned reqs_count = 1000,
                         size_t msg_size = UCS_KBYTE * 70)
     {
-        unsigned warmup_req_count = 5;
+        unsigned warmup_req_count = 1;
```

Comment:
if the number if iterations is 1 now, do we still need a for loop?

Reply:
which one? the PR does not add any new loop, only "fixes" condition for buffers allocation which is correct and type of iterators (usually compilers warn about signed/unsigned comparison).

just noticed leftover in the comment below which must be reverted, so please LMK if I you see smth else to be addressed.